src/mjolnir.py

```diff
@@ -118,9 +118,16 @@ def _on_collect_status(self, event: ops.CollectStatusEvent) -> None:
         if not container.can_connect():
             self._charm.unit.status = ops.MaintenanceStatus("Waiting for pebble")
             return
-        service = container.get_services(MJOLNIR_SERVICE_NAME)
-        if service:
-            logger.debug("%s service already exists, skipping")
+        mjolnir_service = container.get_services(MJOLNIR_SERVICE_NAME)
+        if mjolnir_service:
+            logger.debug("%s service already exists, skipping", MJOLNIR_SERVICE_NAME)
+            return
+        current_services = container.get_services()
+        all_svcs_running = all(svc.is_running() for svc in current_services.values())
+        if not all_svcs_running:
+            # Synapse must be running so the charm can check if there is
+            # a membership_room.
```

Comment:
This comment currently seems to say that it is possible to check if there is a `membership_room` when the charm is not running? The code seems to return so I'm guessing that it isn't possible to check under these conditions

Reply:
Curious, my idea was to clarify that the charm can check for the membership_room only if Synapse is running, otherwise it must wait. I'll rephrase that.